5-infrastructure/business_unit_1/development/variables.tf

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "parent_folder" {
+  type        = string
+  description = "The parent folder or org for environments."
+}
+
+variable "terraform_service_account" {
+  type        = string
+  description = "Service account email of the account to impersonate to run Terraform."
+}
+
+variable "location_primary" {
+  type        = string
+  description = "The primary region for deployment, if not set default locations for each resource are taken from variables file"
+  default     = "us-east1"
+}
+
+variable "location_secondary" {
+  type        = string
+  description = "The secondary region for deployment, if not set default locations for each resource are taken from variables file"
```

Comment:
nit: remove `if not set default locations for each resource are taken from variables file`